### PR TITLE
pkglistgen: Treat stagings differently

### PR DESCRIPTION
**Pull request by coolo**
This is specific to SLE stagings, but won't hurt in openSUSE stagings
as the products there are hand made anyway

### CODE REVIEW

**Comment by coolo** [comment]
these are the current changes I run

File: pkglistgen.py [hunk 1]
```diff
@@ -70,7 +70,7 @@
 SCRIPT_PATH = os.path.dirname(os.path.realpath(__file__))
 ARCHITECTURES = ['x86_64', 'ppc64le', 's390x', 'aarch64']
 DEFAULT_REPOS = ("openSUSE:Factory/standard")
-PRODUCT_SERVICE = '/usr/lib/obs/service/create_single_product'
+PRODUCT_SERVICE = '/home/leaper/obs-service-product_converter/create_single_product'
```
Comment (jberry-suse) [inline pkglistgen.py]: This would seem like a problem.

**Comment by jberry-suse** [comment]
Is the drop list desired for SLE? If so merely configure the `download-baseurl` in `conf.py` to point to something the code can see when running.

File: pkglistgen.py (hunk 1 above)
Comment (coolo) [reply pkglistgen.py]: Just as your code is a problem for me - fair treat, no? :)

**Comment by coolo** [comment]
In general I would say yes, but I would need to experiment with it to see the consequences

File: pkglistgen.py (hunk 1 above)
Comment (jberry-suse) [reply pkglistgen.py]: A joke I presume.

**Comment by coolo** [comment]
I removed the constant for now - to reduce my diff to master

**Pull request merged**